bundles/org.eclipse.jdt.doc.user/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.jdt.doc.user; singleton:=true
-Bundle-Version: 3.15.1700.qualifier
+Bundle-Version: 3.15.1800.qualifier
```

Comment:
this is an unlisted change, but looked to me as a valid change, so included. pls let me know if that is not the case.

Reply:
This may not be required now but will be required by M1. Its good

Reply:
Normally pde environment suggests the version increments when we are updating a plugin. that is when we update the versions.